### PR TITLE
SPOT-233 Sample Arcadia Dashboards for ODM

### PR DESCRIPTION
**Pull request by TaddWood**
"spot_app.json" is a configuration file containing all of the current sample Apache Spot dashboards.  The current set of sample Apache Spot dashboards include: Network Activity, User Activity Summary, User Activity Details, Endpoint Activity Summary, Endpoint Activity Details, and Vulnerabilities views. Details about each of these dashboards can be found in the README along with installation instructions and sources for sample data.  This file can be imported into either Arcadia Instant or Enterprise.  ODM tables should be setup before importing the configuration file.  Once imported the dashboards will align with the ODM tables that have been set up.

screen shot 2017-09-25 at 10 10 47 am
screen shot 2017-09-25 at 10 10 19 am
screen shot 2017-09-25 at 1 30 16 pm
screen shot 2017-09-25 at 10 11 46 am

### CODE REVIEW

File: spot-oa/arcadia/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+spot-setup is licensed under Apache Version 2.0
+
+## Contributing
+
+Create a pull request and contact the maintainers.
+
+## Issues
+
+Report issues at the Apache Spot [issues] (https://github.com/Open-Network-Insight/open-network-insight/issues) page.
```
Comment (natedogs911) [inline spot-oa/arcadia/README.md]: This needs to be changed to https://issues.apache.org/jira/projects/SPOT/issues

Comment (TaddWood) [reply spot-oa/arcadia/README.md]: No problem, just committed the change.  I didn't want to jump the gun assuming what links were correct and which weren't.

**Comment by mhicks-cloudera-com** [comment]
+1

**Comment by natedogs911** [comment]
+1

**Comment by solrac901** [comment]
+1

**Comment by jarcec** [comment]
+1

**Comment by natedogs911** [comment]
merging...

**Comment by TaddWood** [comment]
@natedogs911 I need to update the json file with the dashboards.  There were some last minute modifications requested.  Would you be willing to remerge?

**Pull request merged**